### DISCUSSION TITLE
Any updated resources to learn javascript?

### DISCUSSION THREAD

**Discussion by Lil-Maka** [community/community#136728] [Programming Help] [answered]
### Body

So i want to learn javascript but i don't know where to start, i dont mind if it's english or spanish. 

Thanks!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by andresito1969** [comment, marked as answer]
Hi, if you speak spanish you can learn it here, the course it's brand new. Hope it helps!

https://github.com/mouredev/hello-javascript
